### PR TITLE
fixes for eb30ce5 (new profiles)

### PR DESCRIPTION
**Pull request by glitsj16**
Follow-up fixes for https://github.com/netblue30/firejail/commit/eb30ce54e7a8a75db773a1bbe762a3abdf2ccc42.

### CODE REVIEW

File: etc/profile-a-l/blobby.profile [hunk 1]
```diff
@@ -34,19 +34,20 @@ noroot
 notv
 nou2f
 novideo
-protocol unix,inet,netlink,
+protocol unix,inet,inet6,netlink
 netfilter
```
Comment (rusty-snake) [inline etc/profile-a-l/blobby.profile:38]:
```suggestion
```

File: etc/profile-a-l/funnyboat.profile [hunk 2]
```diff
@@ -7,12 +7,13 @@ include globals.local
...
-include disable-exec.inc
 include allow-python2.inc
 include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
 include disable-interpreters.inc
 include disable-passwdmgr.inc
 include disable-programs.inc
```
Comment (rusty-snake) [inline etc/profile-a-l/funnyboat.profile:19]: ~`include disable-write-mnt.inc`~ (`disable-mnt`)

**Review by rusty-snake — APPROVED** [review]

File: etc/profile-a-l/blobby.profile (hunk 1 above)
Comment (glitsj16) [reply etc/profile-a-l/blobby.profile]: Nice catch! Fixed.

File: etc/profile-a-l/funnyboat.profile (hunk 2 above)
Comment (glitsj16) [reply etc/profile-a-l/funnyboat.profile:19]: Not sure I understand this one. Do you suggest to drop `disable-mnt` and add `include disable-write-mnt.inc`?

Comment (glitsj16) [reply etc/profile-a-l/funnyboat.profile:19]: Doh, now I see what you mean. Fixed!

**Pull request merged**